src/Mobs/Monster.h

```diff
@@ -185,42 +182,40 @@ class cMonster :
 	If no suitable position is found, returns cChunkDef::Height. */
 	int FindFirstNonAirBlockPosition(double a_PosX, double a_PosZ);
 
-	/** Returns if a monster can actually reach a given height by jumping or walking */
-	inline bool IsNextYPosReachable(int a_PosY)
-	{
-		return (
-			(a_PosY <= POSY_TOINT) ||
-			DoesPosYRequireJump(a_PosY)
-			);
-	}
+	/** Returns if the ultimate, final destination has been reached */
+	bool ReachedFinalDestination(void) { return ((m_FinalDestination - GetPosition()).SqrLength() < (m_AttackRange * m_AttackRange)); }
+
+	/** Returns if the intermediate waypoint of m_Destination has been reached */
+	bool ReachedDestination(void) { return ((m_Destination - GetPosition()).SqrLength() < 0.25); }
```

Comment:
Please put this back to 0.5, 0.25 is too precise and the mob will sometimes circle around a waypoint, insisting on landing exactly.